index.js

```diff
@@ -241,6 +241,20 @@ class Executor {
 
         return parsedAnnotations;
     }
+
+    /**
+     * Unzip the ZIP of artifacts
+     * @method unzipArtifacts
+     * @param  {Object} config       Configuration
```

Comment:
need param `config.buildId` like executor-queue.
https://github.com/sonic-screwdriver-cd/executor-queue/blob/30e75c892f5520c4c100d33029ea49572da239a6/index.js#L140

Reply:
Thanks, I didn't understand the param, which isn't refered in this function, also should be written. I'll fix.